.gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules
 .netrc
-moduleIndex.json
\ No newline at end of file
+moduleIndex.json
+
+# MacOS
+.DS_Store
```

Comment:
Intended?

Reply:
I wanted to smuggle it in ... no just kidding.
Anyways let me remove it, it's still waiting with https://github.com/Azure/bicep-registry-modules/pull/647 since 8 weeks now.

Reply:
Maybe it makes sense to reach out to @shenglol directly.